src/Steam.package/STApplicationRootComponent.class/class/initialize.st

```diff
@@ -9,4 +9,15 @@ initialize
 	app
 		addLibrary: JQDeploymentLibrary;
 		addLibrary: JQUiDeploymentLibrary;
-		addLibrary: TBSDeploymentLibrary.
\ No newline at end of file
+		addLibrary: TBSDeploymentLibrary;
+		addLibrary: STZnSSOSeasideDemoFileLibrary.
```

Comment:
We should rename that class later :)

StZnSSOseasideDemoFileLibrary => StSSOSeasideFileLibrary? I'd like to know also what it does? Why is it still called Demo?